test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -232,7 +219,7 @@ def check_fc(existing_schemas):
     if is_fc:
         print("Found forward compatible schemas for all existing schemas")
     else:
-        warnings.warn(
+        print(
```

Comment:
Should this be upgraded to a failure? It's hard to spot prints and warnings.

The error message here is a little goofy. The first sentence is good, but what does it mean to "contact the PyTorch team"? The operators are also listed as "broken" but they are not necessarily broken, they just a potential forward compatibility schema mismatch. We should try to be as clear as possible in our language so developers can understand the issues our test suite presents to them.

Reply:
Changed